README.md

```diff
@@ -36,7 +36,7 @@ Development sponsored by [Caktus Consulting Group, LLC](http://www.caktusgroup.c
 # file: deploy/requirements.yaml
 
 - src: https://github.com/caktus/ansible-role-k8s-web-cluster
-  version: 0.0.7
+  version: 1.0.0
```

Comment:
Yay! Are we planning to use [semantic versioning](https://semver.org/)? If so, can we put a note into the README about that so future devs are sure to update the version in that way.